src/Mod/Path/PathScripts/PathSlotGui.py

```diff
@@ -190,37 +254,37 @@ def updateVisibility(self, sentObj=None):
         if hideFeatures:
             # reset values
             self.CATS = [None, None]
-            self.selectInComboBox('Start to End', self.form.pathOrientation)
+            self.selectInComboBox("Start to End", self.form.pathOrientation)
```

Comment:
I have not looked at how `self.selectInComboBox()` works, but you might need to audit your usage of it to ensure that it's going to work with the new user data.